content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -301,6 +301,16 @@ Below you can find installation instructions for some popular Pod network plugin
 
 {{< tabs name="tabs-pod-install" >}}
 
+{{% tab name="Antrea" %}}
+[Antrea](https://github.com/vmware-tanzu/antrea) is an [Open vSwitch](https://www.openvswitch.org/) based networking and network policy solution. Open vSwitch enables Antrea to implement Kubernetes Network Policies in a high-performance and efficient manner.
+In order Antrea to detect POD network range, you must pass `--pod-network-cidr=192.168.0.0/16` to `kubeadm init`.
```

Comment:
This wording implies that only 192.168.0.0/16 is supported. I assume that Antrea wouldn't make that restriction, as it would be very limiting.

```suggestion
{{< note >}}
If you're using Antrea, you configure a Pod network range by adding a `--pod-network-cidr` parameter to `kubeadm init`.
{{< /note >}}
```

(also, Pod is written “Pod” not “POD”).

Reply:
Thanks @sftim  yes it makes sense, I will change it in a second!